pyproject.toml

```diff
@@ -54,6 +54,7 @@ aleph-alpha-client = {version="^2.15.0", optional = true}
 deeplake = {version = "^3.2.9", optional = true}
 pgvector = {version = "^0.1.6", optional = true}
 psycopg2-binary = {version = "^2.9.5", optional = true}
+boto3 = {version = "^1.26.96", optional = true}
```

Comment:
As you noted on the other PR, we could leave this out - as this is only required for DynamoDB and there is a case to be made that the users should install the necessary database drivers if he/she decides to use one.